hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/hip904/TokenClaimAirdropTest.java

```diff
@@ -117,6 +138,162 @@ final Stream<DynamicTest> claimFungibleTokenAirdrop() {
                         getAccountInfo(RECEIVER).hasToken(relationshipWith(NON_FUNGIBLE_TOKEN)));
     }
 
+    @HapiTest
+    @DisplayName("token claim with no pending airdrop should fail")
+    final Stream<DynamicTest> tokenClaimWithNoPendingAirdrop() {
+        return hapiTest(
+                cryptoCreate(OWNER).balance(ONE_HUNDRED_HBARS).maxAutomaticTokenAssociations(0),
+                cryptoCreate(OWNER_2).balance(ONE_HUNDRED_HBARS).maxAutomaticTokenAssociations(0),
+                cryptoCreate(RECEIVER).balance(ONE_HUNDRED_HBARS),
+                tokenCreate(FUNGIBLE_TOKEN)
+                        .treasury(OWNER)
+                        .tokenType(FUNGIBLE_COMMON)
+                        .initialSupply(1000L),
+                tokenCreate(FUNGIBLE_TOKEN_2)
+                        .treasury(OWNER_2)
+                        .tokenType(FUNGIBLE_COMMON)
+                        .initialSupply(1000L),
+                newKeyNamed(NFT_SUPPLY_KEY),
+                tokenCreate(NON_FUNGIBLE_TOKEN)
+                        .name(NON_FUNGIBLE_TOKEN)
+                        .treasury(OWNER)
+                        .tokenType(TokenType.NON_FUNGIBLE_UNIQUE)
+                        .initialSupply(0L)
+                        .supplyKey(NFT_SUPPLY_KEY),
+                mintToken(NON_FUNGIBLE_TOKEN, List.of(ByteString.copyFromUtf8("a"), ByteString.copyFromUtf8("b"))),
+                tokenClaimAirdrop().hasPrecheck(EMPTY_PENDING_AIRDROP_ID_LIST));
```

Comment:
Do you need all the accounts and tokens created to do claim with empty body, this should be a pureCheck so you can simpify this test to tokenClaimAirdrop().hasPrecheck(EMPTY_PENDING_AIRDROP_ID_LIST)